ports/ptyqt/vcpkg.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "ptyqt",
+  "version-string": "0.6.2",
```

```suggestion
  "version": "0.6.2",
```
This might be served by version.